spec/controllers/branches_controller_spec.rb

```diff
@@ -27,25 +27,25 @@
     context "valid branch name, valid source" do
       let(:branch) { "merge_branch" }
       let(:ref) { "master" }
-      it { should redirect_to("/#{project.path_with_namespace}/tree/merge_branch") }
+      it { is_expected.to redirect_to("/#{project.path_with_namespace}/tree/merge_branch") }
     end
 
     context "invalid branch name, valid ref" do
       let(:branch) { "<script>alert('merge');</script>" }
       let(:ref) { "master" }
-      it { should redirect_to("/#{project.path_with_namespace}/tree/alert('merge');") }
+      it { is_expected.to redirect_to("/#{project.path_with_namespace}/tree/alert('merge');") }
     end
 
     context "valid branch name, invalid ref" do
       let(:branch) { "merge_branch" }
       let(:ref) { "<script>alert('ref');</script>" }
-      it { should render_template("new") }
+      it { is_expected.to render_template("new") }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.